tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -254,7 +254,7 @@ these properties, and if you don't write to them, Godot will optimize away the c
 | in vec4 **FRAGCOORD**                  | Coordinate of pixel center in screen space. ``xy`` specifies position in window, ``z``           |
 |                                        | specifies fragment depth if ``DEPTH`` is not used. Origin is lower-left.                         |
 +----------------------------------------+--------------------------------------------------------------------------------------------------+
-| in bool **FRONT_FACING**               | ``true`` if current face if front face.                                                          |
+| in bool **FRONT_FACING**               | ``true`` if current face is front face.                                                          |
```

Comment:
```suggestion
| in bool **FRONT_FACING**               | ``true`` if current face is front facing.                                                        |
```
Or
```suggestion
| in bool **FRONT_FACING**               | ``true`` if current face is a front face.                                                        |
```

Reply:
Was thinking about making the same change, actually. Thank you!